sync-core/src/test/java/com/cloudant/http/HttpTest.java

```diff
@@ -119,4 +122,52 @@ public void testReadBeforeExecute() throws IOException {
         Assert.assertEquals(bis.available(), data.getBytes().length);
         client.deleteDb();
     }
+
+
+    //NOTE: This test doesn't work with specified couch servers,
+    // the URL will always include the creds specified for the test
+    //
+    // A couchdb server needs to be set and running with the correct
+    // security settings, the database *must* not be public, it *must*
+    // be named cookie_test
+    //
+    @Test
+    public void testCookieAuthWithoutRetry() throws IOException {
```

Comment:
Stupid me, that's what happens when you only look at the diff and not the whole file.